examples/example4/code_surf.m

```diff
@@ -0,0 +1,30 @@
+close all
```

Comment:
Whatever this is, it needs a comment.

Reply:
Actually this file should not be included. Removed.